ResearchKit/Common/ORKPasscodeStepViewController.m

```diff
@@ -39,9 +39,14 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 #import <AudioToolbox/AudioToolbox.h>
 #import <LocalAuthentication/LocalAuthentication.h>
 
+static CGFloat const kForgotPasscodeVerticalPadding     = 50.0f;
```

Comment:
do these hardcoded values work on all supported screen sizes? would rather handle with constraints but given time pressure will accept this if it works...